Find a functionally equivalent code:android.view.accessibility.AccessibilityEvent.getRecordCount:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the number of records contained in the event.
 *
 * @return The number of records.
 */

@@ -1,3 +1,3 @@
 {
-    return mRecords.size();
+    return mRecords == null ? 0 : mRecords.size();
 }

